### DISCUSSION TITLE
Multiple different servers into 1 PLAN database?

### DISCUSSION THREAD

**Discussion by trentnbauer** [plan-player-analytics/Plan#3605] [Q&A] [answered]
Hi all,
I host multiple servers and would love to collate player data into 1 PLAN instance.
For example, I have a
- Spigot server
- 2 forge modded servers

I would love to be able to see player data from all 3 servers in the 1 PLAN instance. Is this possible, and how do I configure this? What issues could arise from this?

**Comment by AuroraLS3** [comment, marked as answer]
Just point them all at the same MySQL database.

You can follow this step even if you don't have Bungee/Velocity running
https://github.com/plan-player-analytics/Plan/wiki/Bungee-Set-Up#step-7-set-up-plan-on-game-servers
